src/dlgprefautodj.cpp

```diff
@@ -66,12 +89,16 @@ void DlgPrefAutoDJ::slotApply() {
 void DlgPrefAutoDJ::slotResetToDefaults() {
```

Comment:
No, we should allow use case: 
* Look 
* Reset to default 
* Dislike 
* Cancel